examples/directml/stable_diffusion_xl/config_vae_decoder.json

```diff
@@ -75,7 +75,43 @@
                     "enable_packed_kv": true,
                     "enable_bias_add": false
                 },
-                "force_fp32_ops": ["RandomNormalLike"]
+                "force_fp32_ops": ["RandomNormalLike"],
+                "force_fp32_nodes": [
+                    "/decoder/up_blocks.2/upsamplers.0/conv/Conv",
```

Comment:
Starting from the Conv node, output is overflow by casting to float16. Wrapping this single node also causing overflow, so all the nodes after it need to be skipped.